docs/application/dotnet/guides/app-management/widget-viewer.md

```diff
@@ -0,0 +1,129 @@
+
+```csharp
+using Tizen.NUI;
+using Tizen.NUI.BaseComponents;
+```
+
+Commonly the `OnCreate()` callback is implemented to initialize UI of an application:
+
+```csharp
+namespace NUIWidgetViewer
+{
+    class Program : NUIApplication
+    {
+        protected override void OnCreate()
+        {
+            base.OnCreate();
+            Initialize();
+        }
+    }
+    //...
+```
+
+In this case, the `Initialize()` function called from the `OnCreate()` callback:
+
+- Creates the `widgetsList` View instance and:
+    - Set its size using `WidthResizePolicy` and `HeightResizePolicy` to screen size.
+    - Setup horizontal `Layout` with padding and margin.
+- Creates the `widgetViewManager` instance and register current application as a viewer for installed widgets using [Application ID](https://samsung.github.io/TizenFX/latest/api/Tizen.Applications.ApplicationInfo.html)
+- Creates instances of two widgets. The `galleryWidget` and `contactsWidget` are created using `widgetViewManager` object.
+- Inserts created widgets into created `widgetsList` and add `widgetsList` to the default layer of the application window.
+
+```csharp
+    void Initialize()
+    {
+        View widgetsList = new View()
+        {
+            WidthResizePolicy = ResizePolicyType.FillToParent,
+            HeightResizePolicy = ResizePolicyType.FillToParent,
+            Layout = new LinearLayout()
+            {
+                LinearAlignment = LinearLayout.Alignment.CenterVertical,
+                LinearOrientation = LinearLayout.Orientation.Horizontal,
+                CellPadding = new Size2D(20, 20),
+                Padding = new Extents(30, 30, 30, 30)
+            }
+        };
+
+        WidgetViewManager widgetViewManager = new WidgetViewManager(this, this.ApplicationInfo.ApplicationId);
+
+        var galleryWidget = widgetViewManager.AddWidget("org.tizen.gallery.widget", "", 
+                Window.Instance.WindowSize.Width / 2 - 40, Window.Instance.WindowSize.Height, 0);
+        var contactsWidget = widgetViewManager.AddWidget("org.tizen.contacts.widget", "", 
+                Window.Instance.WindowSize.Width / 2 - 40, Window.Instance.WindowSize.Height, 0);
+        
+        widgetsList.Add(galleryWidget);
+        widgetsList.Add(contactsWidget);
+
+        Window.Instance.GetDefaultLayer().Add(widgetsList);
+    }
+```
+
+Handling back button press events:
+
+```csharp
+    public void OnKeyEvent(object sender, Window.KeyEventArgs e)
+    {
+        if (e.Key.State == Key.StateType.Down && (e.Key.KeyPressedName == "XF86Back" || e.Key.KeyPressedName == "Escape"))
+        {
+            Exit();
+        }
+    }
+```
+
+Main code of the application:
+
+```csharp
+    static void Main(string[] args)
+    {
+        var app = new Program();
+        app.Run(args);
+    }
+```
+
+For full source code of the example. see [here](./source-code/widget-viewer.cs).
+
+## Related information
+  - Dependencies
+    -   Tizen 4.0 and Higher
```

Comment:
Hmm I set the title of this PR to inform about it is requested to merge it to 6.5 branch. Dependency I think is ok. I think it will be easier to merge tizen_6.5_prepare branch without conflicts

Reply:
If you can confirm that widget viewer was supported since 4.0, it is okay to keep current state.

Reply:
Sure, please check following link: 

https://docs.tizen.org/application/dotnet/api/TizenFX/API4/api/Tizen.NUI.WidgetViewManager.html

All classes used in snippets are available in Tizen API 4

Reply:
I see. Thank you for checking & confirming.